nixos/modules/services/desktops/gnome/gnome-remote-desktop.nix

```diff
@@ -26,8 +26,20 @@ with lib;
   ###### implementation
   config = mkIf config.services.gnome.gnome-remote-desktop.enable {
     services.pipewire.enable = true;
+    services.dbus.packages = [ pkgs.gnome.gnome-remote-desktop ];
+
+    environment.systemPackages = [ pkgs.gnome.gnome-remote-desktop ];
 
     systemd.packages = [ pkgs.gnome.gnome-remote-desktop ];
     systemd.tmpfiles.packages = [ pkgs.gnome.gnome-remote-desktop ];
+
+    users = {
```

Comment:
Ooh, I didn't even know that NixOS even had that feature! Would you recommend that I swap out the manual user and group creation for `systemd.sysusers.enable = true`?

Reply:
Should it be conditional based on `sysusers` being enabled? Do we do that anywhere right now - and should we? Afaict we only have a comment next to flatpak to switch to sysusers at some point when possible.

Reply:
> Would you recommend that I swap out the manual user and group creation for `systemd.sysusers.enable = true`?

Yeah, that could have unintended side-effects, as you found out. Stuff like this is usually better enabled top-down, rather than bottom up.

> Should user creation be conditional based on `sysusers` being enabled? Do we do that anywhere right now - and should we?

Not sure how it behaves when there are conflicts between `sysusers` and `users.users`. It might be a good idea to test it for future-proofing once the `sysusers.nix` functionality is enabled but not critical at the moment.

Reply:
Actually, looking at the `sysuser.nix` that just makes it so `users.users` users are created using `sysusers.d` – the systemd functionality should [already be enabled](https://github.com/NixOS/nixpkgs/pull/264879).

Reply:
I don't think that has any effect beyond being built with necessary support, same as homed support always being compiled in but not on by default - but I'd prefer if maybe someone who maintains systemd in nix could weigh in.

Maybe for now it's enough to just add a comment in the module saying that the package provides a sysusers.d file we should switch to one day.